gatsby-node.js

```diff
@@ -28,64 +28,6 @@ exports.onCreateNode = ({ node }) => {
   return node;
 };
 
-exports.createSchemaCustomization = ({ actions }) => {
```

Comment:
Could you expand why this code is needed to allow optional images? I ask because a have an optional image inter alia for the skill icon and there are no problems

Reply:
So try deleting all icon images, with optional images there was an issue when non of them were provided, gatsby query was failing